docs/configuration/settings.rst

```diff
@@ -434,6 +415,48 @@ values for OAuth2 client settings.
 See :github:`mozilla-services/cliquet-fxa`.
 
 
+.. _configuring-notifications:
+
+Notifications
+=============
+
+*Kinto* has a notification system, and the event listeners are configured using
+the *event_handlers* setting, which takes a list of aliases.
```

Comment:
The name of the listener that you then configure.
This is working as for logging configuration in python where you name your handlers before using them and configuring them.